### DISCUSSION TITLE
Beyond UI: Using Slint with C++ libs

### DISCUSSION THREAD

**Discussion by aurindam** [slint-ui/slint#4106] [Blog Comments]
Blog post - https://slint.dev/blog/beyond-ui-using-slint-with-cpp

**Comment by GayarUsmanov** [comment]
Hi @aurindam,

Do you happen to know the timeline for developing of this feature?
[https://slint.dev/blog/beyond-ui-using-slint-with-cpp#:~:text=3D%20rendering%20inside%20Slint%20UI%20using%20KDGpu](https://slint.dev/blog/beyond-ui-using-slint-with-cpp#:~:text=3D%20rendering%20inside%20Slint%20UI%20using%20KDGpu)

or maybe just a general update on 3D rendering in Slint?
